epitator/count_annotator.py

```diff
@@ -196,29 +184,16 @@ def search_lemmas(lemmas, match_name=None):
             max_dist=50)
         # Add singular case reports
         singular_case_spans = []
-        determiner_lemmas = set(['a', 'an', 'the'])
+        determiner_lemmas = set(['a', 'an', 'the', 'one'])
```

Comment:
Would be interesting to test 'each' here also.

Reply:
What context were you thinking of "each" appearing in? I'm not seeing how that would apply to singular case reports.